apps/src/templates/SchoolDataInputs.jsx

```diff
@@ -0,0 +1,91 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import i18n from '@cdo/locale';
+import {Heading1, Heading2, Heading3} from '@cdo/apps/lib/ui/Headings';
```

Comment:
I think you want to use the Headings from the typography component: https://github.com/code-dot-org/code-dot-org/tree/staging/apps/src/componentLibrary/typography. Now I'm doubting which is right but I hate that there are multiple 😬

Reply:
Oh dang hahah okay we've moved to T-shirt sizing then! Going to redo all of them 
<img width="351" alt="Screenshot 2024-02-27 at 3 40 18 PM" src="https://github.com/code-dot-org/code-dot-org/assets/37230822/a9b9ae9e-030e-4df6-8084-59c7a858784c">

Reply:
Okay but now I'm seeing folks import 'Heading1' from that file... I juuuust didn't understand how the scss was working